GitUI/CommitInfo/CommitInfo.Designer.cs

```diff
@@ -93,16 +104,18 @@ private void InitializeComponent()
             this.rtbxCommitMessage.BackColor = System.Drawing.SystemColors.Control;
             this.rtbxCommitMessage.BorderStyle = System.Windows.Forms.BorderStyle.None;
             this.rtbxCommitMessage.ContextMenuStrip = this.commitInfoContextMenuStrip;
-            this.rtbxCommitMessage.Location = new System.Drawing.Point(8, 8);
-            this.rtbxCommitMessage.Margin = new System.Windows.Forms.Padding(8);
+            this.rtbxCommitMessage.Location = new System.Drawing.Point(10, 10);
+            this.rtbxCommitMessage.Margin = new System.Windows.Forms.Padding(10, 10, 10, 10);
             this.rtbxCommitMessage.Name = "rtbxCommitMessage";
+            this.rtbxCommitMessage.ReadOnly = true;
```

Comment:
Could you revert all other changes, and only commit this change? All the other changes make it harder to review. For example, the location and margin are also changed. Was this intended?

Reply:
I just hit save in the WinForms designer in the first commit. But okay, I'll drop this commit.

Reply:
@spdr870 Done.

Reply:
@ulrichb we are cautious about automated designer changes as they can regress high-DPI support, and it's hard to verify whether that happens by just looking at the diff.